system/gameModes/default/compatibilityDataControl.lst

```diff
@@ -36,4 +36,5 @@ DEFAULTVARIABLEVALUE:BOOLEAN|False
 DEFAULTVARIABLEVALUE:STRING|
 DEFAULTVARIABLEVALUE:DICE|0
 DEFAULTVARIABLEVALUE:ALIGNMENT|None
+DEFAULTVARIABLEVALUE:HANDED|Right
```

Comment:
does this require data approval?

Reply:
Not particularly, it's extracting it out of the existing code (hardcoded)